vmi.py

```diff
@@ -0,0 +1,280 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import seaborn as sns
+
+# %%
+islaidos=pd.read_csv("Gyventojuislaidos.csv")
```

Comment:
```suggestion
islaidos = pd.read_csv("data/datasets/gov/vmi/gyventoju_islaidos/Gyventojuislaidos.csv")
```

Visi duomenys turėtu būti sudėti į `data/` katalogą, kuris yra įtrauktas į `.gitignore` ir kelias iki duomenų failo ir jo pavadinimas turi sutapti su duomenų struktūros apraše nurodyta vardų erdve ir modelio pavadinimu.

Reply:
Pamiršau, dar paminėti, kad ir patys `.py`, `.ipynb` failai turėtu keliauti į katalogus, kurie atitinka duomenų rinkinio vadų erdvės pavadinimą.